dubbo-common/src/main/java/org/apache/dubbo/common/compiler/support/JdkCompiler.java

```diff
@@ -61,14 +61,22 @@
 
     private final JavaFileManagerImpl javaFileManager;
 
-    private volatile List<String> options;
+    private final List<String> options;
 
-    public JdkCompiler() {
-        options = new ArrayList<String>();
-        options.add("-source");
-        options.add("1.6");
-        options.add("-target");
-        options.add("1.6");
+    private static final String DEFAULT_JAVA_VERSION = "1.6";

```

Comment:
shall we upgrade this to 1.8 ?

Reply:
@AlbumenJ 

Well in my own usecases I need 1.8, so I want such a mechanism to configure it.

However I really do not know why you originally chose 1.6 here.

Is there any special reason?

I myself will not stop you from change it to 1.8, but I really don't know about other users of this class's opinions.

Reply:
For Dubbo 2.7.x, jdk 1.6 is not compatiable.

Reply:
> For Dubbo 2.7.x, jdk 1.6 is not compatiable.

@AlbumenJ sounds reasonable.
OK, then we change it to 1.8.